configs/components/puppet.rb

```diff
@@ -59,6 +63,83 @@
     pkg.install_service "ext/solaris/smf/puppet.xml", "ext/solaris/smf/puppet", service_type: "network"
   when "aix"
     pkg.install_service "resources/aix/puppet.service", nil, "puppet"
+  when "windows"
+    description = "Periodically fetches and applies configurations from a Puppet master server."
+    service_hash = [
```

Comment:
@Iristyle if we move these hashed to a file, vanagon still has to be aware of wix and how to handle services... Are you saying that rather than have a template we should hard code the services into files? I think Michael's suggestion was to move these hashes into a file, and then read in the hash and pass it to the `install_service` command as it currently happening